frontend/src/components/WindSettings.jsx

```diff
@@ -20,391 +19,209 @@ const useStyles = makeStyles((theme) => ({
-                                variant="outlined"
-                                type="number"
-                                onChange={handleWindChange}
-                                value={envConf.Wind.Force}
-                                inputProps={{ min: 0 }}
-                                fullWidth
-                            />
-                        </Tooltip>
-                    </Grid>
-                </Grid>
 
-            {(selectedWindType === "Turbulent Wind" || selectedWindType === "Wind Shear")  && (
-                    <Grid item container alignItems="center" direction="row">
-                        <Grid item xs={4}>
-                            <InputLabel id="fluctuation-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
-                                Fluctuation %
-                            </InputLabel>
-                        </Grid>
-                        <Grid item xs={6}>
-                            <Tooltip title="Enter Fluctuation %" placement='bottom'>
-                                <TextField id="Fluctuation %" fullWidth
-                                    variant="outlined" 
-                                    type="number"
-                                    sx = {{
-                                        backgroundColor: '#F5F5DC',
-                                        '& .MuiOutlinedInput-root': {
-                                            '& .MuiInputBase-input': {
-                                                padding: '6px 8px',
-                                            },
-                                        },
-                                    }}
-                                    onChange={handleFLuctuationChange} 
-                                    value={fluctuationPercentage} 
-                                    inputProps={{ min: 0, 
-                                        max: 100, 
-                                        step: 0.1 
-                                    }} 
-                                />
-                            </Tooltip>
+                    {windBlocks.map((windBlock, index) => (
+                        <Grid item container spacing={2} xs={12} classes={{ root: classes.backdropFilter }} key={index}>
+                            {renderSelectField("Wind Type", windBlock.windType, (e) =>
+                                setWindBlockData(index, { windType: e.target.value }), WindType)}
+                            {renderSelectField("Wind Direction", windBlock.windDirection, (e) =>
+                                setWindBlockData(index, { windDirection: e.target.value }), WindDirection)}
+                            {renderTextField("Wind Velocity (m/s)", windBlock.windVelocity, (e) =>
+                                setWindBlockData(index, { windVelocity: e.target.value }), { min: 0 })}
+                            {renderTextField("Fluctuation %", windBlock.fluctuationPercentage, (e) =>
```

Comment:
@syedomair0 `Fluctuation` field should be displayed only when `Turbulent Wind` Type is selected